app/profile/SkillsTab.tsx

```diff
@@ -21,8 +32,48 @@ export const SkillsTab = () => {
     setModel({ ...model!, skills: encodeSkillSelection(selection) });
   };
 
+  const enableSubmitButton = bioState.submissionState === "changes_done";
+
+  const submitButtonLabel =
+    bioState.submissionState === "submitted_successfully"
+      ? "Úspěšně uloženo 🎉"
+      : "Uložit změny";
+
   return (
     <section className="mb-10">
+      <p className="mb-4">Řekni něco málo o sobě, ať tě lidé lépe poznají</p>
+      <textarea
+        className="block w-full max-w-prose rounded-md border-2 border-gray p-2"
+        rows={3}
+        defaultValue={actualBio}
+        onChange={(e) =>
+          setBioState(() => ({
+            submissionState: "changes_done",
+            bio: e.target.value,
+          }))
+        }
+        placeholder="Čemu se věnuješ? Co tě baví? Do jakých projektů tě láká se zapojit?"
+      />
+
+      <button
+        onClick={() => {
+          setModel({
```

Comment:
Ta čárka mezi `setModel` a `setBioState` je omyl, ne? Protože je to JavaScript, tak to funguje :), ale podle mě myslíš tohle?

```typescript
setModel({
    ...model!,
    bio: bioState.bio,
});
setBioState((prevState) => ({
    ...prevState,
    submissionState: "submitted_successfully",
}));
```

A to druhé volání by se dalo zjednodušit ještě takhle:

```typescript
setBioState({
    ...bioState,
    submissionState: "submitted_successfully",
});
```